services/brig/src/Brig/API/User.hs

```diff
@@ -191,6 +189,10 @@ createUser new@NewUser{..} = do
                 return (True, Just $ CreateUserTeam (Team.inTeam inv) nm)
         Nothing -> return (False, Nothing)
 
+    case (ident, tid) of
```

Comment:
Note that here the user also joins a team (notice the `joinedTeam` variable down below)

Reply:
You mean I'm not doing that, but should?  Will check!

Reply:
> You mean I'm not doing that, but should? Will check!

Yes ;)